neqo-bin/src/bin/client/main.rs

```diff
@@ -451,7 +452,7 @@ async fn main() -> Res<()> {
     neqo_common::log::init(Some(args.verbose.log_level_filter()));
     args.update_for_tests();
 
-    init();
+    init().map_err(|_| ClientError::InternalError)?;
```

Comment:
```suggestion
    init()?;
```

Then, you can implement `From<neqo_crypto::Error> for ClientError`.

The same applies for the server code, though you can't get `io::Error`.